internal/cache/file/cache_handler_test.go

```diff
@@ -71,7 +73,7 @@ func (chrT *cacheHandlerTest) SetUp(*TestInfo) {
 	// Create test object in the bucket.
 	testObjectContent := make([]byte, TestObjectSize)
 	_, err := rand.Read(testObjectContent)
-	AssertEq(nil, err)
+	assert.NoError(chrT.T(), err)
```

Comment:
Should you use require.NoError to immediately abort test execution if there is an error? Otherwise, despite the error, the next set of statements would continue getting executed.  Please review other usages as well.

require.* methods specify precondition - something that must be satisfied unless there is something wrong with the tests. However, for tests that don't follow the Arrange-Act-Assert pattern e.g. have multiple assert blocks, it's not clear what's the precondition and what should be asserted. So, I guess, we can leave such cases as is and fix them once they are refactored into smaller tests.

Reply:
I was using require.* in the discarded PR that I created. It will require me to go to each assert again and make that change. 

I have been making that change in the next PR (because that is easier for me using the discarded PR but anyways I will include here only, will take some time to do that)